.gitignore

```diff
@@ -160,4 +160,5 @@ cython_debug/
 #.idea/
 
 Captions/
-savedData/
\ No newline at end of file
+savedData/
+Output Data/
```

Comment:
I can see Q generation in simple text. If I remember correctly there was a request to list all the topics generated from the Bert. Is that going to be separate issue?

Reply:
I guess #25 handle that. I will resolve this comment